zubhub_frontend/zubhub/src/views/create_project/script.js

```diff
@@ -693,21 +693,8 @@ export const validationSchema = Yup.object().shape({
     //       ? false
     //       : true;
     //   }),
-    materials_used: Yup.string()
-        .max(10000, 'max')
-        .test('empty', 'required', value => {
-            let is_empty = true;
-
-            value &&
-                value.split(',').forEach(material => {
-                    if (material) {
-                        is_empty = false;
-                    }
-                });
-
-            return !is_empty;
-        }),
-    category: Yup.string(),
+    materials_used: Yup.array(Yup.string()).required(),
+    category: Yup.array(),
```

Comment:
this too

Reply:
Everything works fine really, I presented these changes in the call

Reply:
see this https://github.com/unstructuredstudio/zubhub/pull/1089#discussion_r1477551368